app/views/settings/cookie-policy.html.erb

```diff
@@ -1,8 +1,18 @@
 <div class="govuk-width-container">
-      <div class="govuk-grid-row govuk-!-margin-bottom-7">
-        <%== insert_block('cookies') %>
+  <div class="govuk-grid-row govuk-!-margin-bottom-7">
+    <% if notice %>
+      <div class="govuk-success-summary banner-with-tick" data-qa="cookie-consent-success-message" aria-labelledby="success-message" tabindex="-1" role="alert" data-cookie-confirmation="true">
+        <div class="govuk-success-summary__title">
+          <h2 class="govuk-heading-m" id="success-message">
+            <%= notice.html_safe %>
+          </h2>
+        </div>
       </div>
```

Comment:
Can we not do this?  I was fixing this in the other bug by putting it in the layout.  Please don't re-introduce it.

Reply:
Sure, I was going to sort it in the merge, so Darren does not raise the error again, but I will delete it

Reply:
Done